js/form.js

```diff
@@ -0,0 +1,16 @@
+const form = document.getElementById('form');
+const email = document.getElementById('email_form');
+const error = document.querySelector('.error-email');
+const regex = /^[a-z0-9_-]+@[a-z0-9]+\.[a-z]+\.?[a-z]+/g;
+
+form.addEventListener('submit', (e) => {
+  if (regex.test(email.value)) {
+    error.textContent = '';
+    email.style.border = '1px solid #cfd8dc';
+  } else {
+    e.preventDefault();
+    email.style.border = '3px solid red';
+    error.textContent = 'The email should be lowercase.';
+    error.style.color = 'red';
+  }
+});
```

Comment:
[OPTIONAL] I think we could wrap all of our JS code in a `validateForm` function. And the call `validateForm` once the web app is finished loading. This will ensure that it works correctly. Imagine that the script loads, but maybe some of the elements on the document aren't ready yet. Then it could fail to select these.

Start the script running once the page has finished loading like this: `window.onload = validateForm`

Reply:
Done